### ISSUE TITLE
Get access to DCMS UR report summary

### ISSUE THREAD

**Issue by JohnLewisUR** [GSS-Cogs/DCMS#14]
On a call with Penny and Rory they mentioned that DCMS had a UR consultancy take a look at their outputs and were waiting on the report. The report doesn't cover their specific output, but the findings may still apply and will still be useful to our project more widely.
In the first instance, should be able to get access to the summary/headline findings, as the full report has not been released yet.

**Comment by JohnLewisUR** [comment]
Have emailed Rory today to chase this up. will update when i receive a response. 
We may also want to add another card for getting access to the full report when that is ready, as it'll be helpful.

**Comment by JohnLewisUR** [comment]
Have received the [overview of the report](https://docs.google.com/document/d/11ilsWthGuMSXT8srve7DkOdbeuB3WKmg/edit?usp=sharing&ouid=116884451069231315641&rtpof=true&sd=true) from Rory. From my original conversation with Rory and Penny the full report isn't available yet
